theme/keywind/login/logout-confirm.ftl

```diff
@@ -9,7 +9,7 @@
     <p>${msg("logoutConfirmHeader")}</p>
     <form action="${url.logoutConfirmAction}" class="m-0 space-y-4" method="post">
       <input name="session_code" type="hidden" value="${logoutConfirm.code}">
-      <@buttonPrimary.kw type="submit">
+      <@buttonPrimary.kw name="confirmLogout" type="submit" value="Logout">
```

Comment:
```suggestion
      <@buttonPrimary.kw name="confirmLogout" type="submit" value="${msg("doLogout")}">
```

Thanks for your pull request. This value matches the original template.

https://github.com/keycloak/keycloak/blob/main/themes/src/main/resources/theme/base/login/logout-confirm.ftl#L20

Reply:
Thanks for your comment!

Using `value="${msg("doLogout")}"` leads to templating errors, so I used `value="${msg('doLogout')}"` and it's working fine!